app/views/application/_show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="col-lg-12">
+
+<h1><%= @media.name %> <small><%= header %>: <%= name %></small></h1>
+
+<h4>Ranked: <%= @media.ranking %></h4>
+
+<p><%= @media.description %></p>
+
+<%= button_to "Upvote", polymorphic_path(@media, :action => :upvote),
```

Comment:
In the `_index` partial template, we have a `link_to` for the upvote option. Is there a reason we're using `button_to` here instead? And does that choice relate to the need to specify `:method => :patch` on this line as well?